spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindSqlInjection.java

```diff
@@ -259,6 +266,15 @@ private boolean isStringAppend(Instruction ins, ConstantPoolGen cpg) {
         return false;
     }
 
+    private boolean isJava9AndAboveStringAppend(Instruction ins, ConstantPoolGen cpg) {
+        if (ins instanceof INVOKEDYNAMIC) {
```

Comment:
This could all be one line.

    return ins instanceof INVOKEDYNAMIC
        && "makeConcatWithConstants".equals(((INVOKEDYNAMIC) ins).getMethodName(cpg));